src/remediations/fifi.integration.js

```diff
@@ -1166,7 +1166,7 @@ describe('FiFi', function () {
                                 display_name: null
                             }
                         ],
-                        type: 'Satellite',
+                        type: 'satellite',
```

Comment:
this looks like potentially breaking - e.g. frontend may be expecting the previous value?

Reply:
@jharting I checked the Frontend code for fifi and it looks like they're looking for a lowercase 'satellite' so I believe this should be alright.